Tutorials for lab_dev and physics (#392)
XanaduAI/MrMustard: doc/code/physics/fock.rst

The Fock representation theory
==========================================

.. currentmodule:: mrmustard.physics.fock

.. automodapi:: mrmustard.physics.fock
    :no-heading:
    :include-all-objects:

> This section of functions is the helper function section to calculate properties of object in Fock representation, which needs to be rearranged and possibly be moved under Fock representation class.

Reviewer: ```suggestion
This module contains helper functions to calculate properties of objects in Fock representation.
```
